src/app/shared/components/fy-userlist/fy-userlist-modal/fy-userlist-modal.component.ts

```
@@ -226,17 +214,14 @@
               };
-              const newArr = [];
+              const newArr: (Partial<Employee> & { isNew?: boolean })[] = [];
               newArr.push(newItem);
               newlyAddedItems = newArr.concat(newlyAddedItems);
               return newlyAddedItems.filter(
                 (item) =>
-                  item &&
-                  item.us_email &&
-                  item.us_email.length > 0 &&
-                  item.us_email.toLowerCase().includes(searchTextLowerCase)
+                  item && item.email && item.email.length > 0 && item.email.toLowerCase().includes(searchTextLowerCase)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)